data/showcase/leximpact_socio_fiscal.yml

```diff
@@ -10,5 +12,13 @@ author:
   contact: leximpact@assemblee-nationale.fr
   link: https://leximpact.an.fr/
 source:
-  repository: https://git.leximpact.dev/leximpact/leximpact-socio-fiscal-ui
+  repository: https://git.leximpact.dev/leximpact/leximpact-socio-fiscal-api
+  # web api for use case calculation: https://git.leximpact.dev/leximpact/leximpact-socio-fiscal-api
+  # web api for state budget calculation: https://git.leximpact.dev/leximpact/leximpact-socio-fiscal-simu-etat
+  # web interface: https://git.leximpact.dev/leximpact/leximpact-socio-fiscal-ui
+  # customize models components for interface: https://git.leximpact.dev/leximpact/leximpact-socio-fiscal-openfisca-json
+  # detect models components: https://git.leximpact.dev/leximpact/openfisca-json-model
+  # adapt openfisca_france_data database for state budget with openfisca scenario: https://git.leximpact.dev/leximpact/leximpact-prepare-data
+  # link and clean up france survey for state budget: https://github.com/openfisca/openfisca-france-data
+  # common data tools for surveys: https://github.com/openfisca/openfisca-survey-manager
```

Comment:
Optional but here is the UI repository + other dependencies directly linked to openfisca 😅